nixos/modules/services/networking/pdns-recursor.nix

```diff
@@ -172,46 +169,26 @@ in {
       dnssec           = cfg.dnssecValidation;
       serve-rfc1918    = cfg.serveRFC1918;
       lua-config-file  = pkgs.writeText "recursor.lua" cfg.luaConfig;
-
```

Comment:
Is there a reason for this change? I dont particularly like adding options to the command line.

Reply:
The upstream unit passes these options as command line arguments, not in the settings file. I assume upstream passes these 4 options (`daemon`, `write-pid`, `disable-syslog`, and `log-timestamp`) as command line arguments because with `systemd` these are really the only values that make practical sense :man_shrugging:

Reply:
I think it's better to keep configure them from `pdns-recursor.settings`,  to make them easier to override by end users.